extensions/ibox/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "ibox",
+  "version": "0.1.1",
+  "title": "搜索书签",
+  "description": "i box",
+  "icon": "command-icon.png",
+  "author": "aidenZou",
+  "categories": [
+    "Applications"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "iBox",
+      "description": "用于快捷搜索书签收藏并打开",
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

Reply:
done